docs/snippets/collecting-metrics.md

```diff
@@ -47,27 +47,71 @@ aggregating timeseries metrics and alerting. It can also be used to scrape the O
     kubectl apply -f https://raw.githubusercontent.com/knative-extensions/monitoring/main/servicemonitor.yaml
     ```
 
-1. Grafana dashboards can be imported from the [`knative-extensions` repository](https://github.com/knative-extensions/monitoring/tree/main/grafana).
+### Access the Prometheus instance locally
+
+By default, the Prometheus instance is only exposed on a private service named `prometheus-operated`.
+
+To access the console in your web browser:
+
+1. Enter the command:
+
+    ```bash
+    kubectl port-forward -n default svc/prometheus-operated 9090
+    ```
+
+1. Access the console in your browser via `http://localhost:9090`.
+
+### Setting up Grafana
+
+1. Grafana dashboards can be imported from the [`monitoring` repository](https://github.com/knative-extensions/monitoring/tree/main/grafana).
 
 1. If you are using the Grafana Helm Chart with the Dashboard Sidecar enabled, you can load the dashboards by applying the following configmaps.
 
     ```bash
     kubectl apply -f https://raw.githubusercontent.com/knative-extensions/monitoring/main/grafana/dashboards.yaml
     ```
 
-### Access the Prometheus instance locally
+    !!! caution
+        You will need to ensure that the helm chart has following values configured, otherwise the dashboards loading will not work.
+        ```yaml
+        grafana:
+          sidecar:
+            dashboards:
+              enabled: true
+              searchNamespace: ALL
+        ```
+        If you have an existing configmaps check the value of `grafana_dashboard` label. In case this is `true`, add the `labelValue: true` attribute to the helm chart above.
+        ```bash
+        kubectl -n knative-serving get configmaps/knative-serving-dashboards -o jsonpath="{.metadata.labels.grafana_dashboard}"; echo
```

Comment:
How can I adapt namespace and cm name based on the selected section? (Serving/Eventing)

Reply:
Let's omit this part about printing that value. Also let's be specific where to add the value. eg after `searchNamespace: ALL`.